test/integration/liquid/cache.ts

```diff
@@ -175,4 +190,19 @@ describe('LiquidOptions#cache', function () {
       expect(y).to.equal('foo')
     })
   })
+  it('should respect passed in cache=false option', async function () {
```

Comment:
seems in the wrong level, should be inside describe renderFileSync

Reply:
Sorry about that, fixed